qiskit_ibm_runtime/options/zne_options.py

```diff
@@ -34,7 +34,52 @@
 
 @primitive_dataclass
 class ZneOptions:
-    """Zero noise extrapolation mitigation options. This is only used by V2 Estimator.
+    """Zero noise extrapolation mitigation options. This is only used by the V2 Estimator.
+
+    ..note::
+
+        Any V2 estimator is guaranteed to return data fields called ``evs`` and ``stds`` that
+        report the desired expectation value estimates and errors, respectively.
+        When ZNE options are enabled in the runtime estimator, additional data is returned.
+
+        In particular, suppose an input pub has observable array shape ``obs_shape`` and parameter
+        values shape ``par_shape``, with corresponding pub shape ``shape``. Then the corresponding
```

Comment:
```suggestion
        values shape ``par_shape``, with corresponding pub shape ``shape = obs_shape * par_shape``. Then the corresponding
```

Reply:
we'd actually want ``shape = numpy.broadcast_shapes(obs_shape, par_shape)``. Would this be okay?